dietpi/dietpi-software

```diff
@@ -13330,6 +13471,21 @@ _EOF_
 			rm /etc/systemd/system/spotify-connect-web.service
 
 		fi
+    
+    	software_id=182 # Unbound
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
+
+			Banner_Uninstalling
+
+			G_AGP unbound
+      
+			if grep -q 'PIHOLE_DNS_1=127.0.0.1#5353' /etc/pihole/setupVars.conf
```

Comment:
We should check first if that file actually exist, else an error message is faced.